src/pykeen/evaluation/rank_based_evaluator.py

```diff
@@ -72,6 +73,7 @@
     RANK_STD: np.std,
     RANK_VARIANCE: np.var,
     RANK_MAD: stats.median_abs_deviation,
+    RANK_COUNT: lambda x: np.asarray(x.size),
```

Comment:
this might need a corresponding entry in the metric rank class

Reply:
can you add a test to one of the pipeline tests that this result is actually in the stuff that comes out

Reply:
>this might need a corresponding entry in the metric rank class
done in f1e49bf 

> can you add a test to one of the pipeline tests that this result is actually in the stuff that comes out
done in  2a1195f

Reply:
danke dir!